filebeat/etc/beat.full.yml

```diff
@@ -176,6 +176,11 @@ filebeat.prospectors:
   # Note: Potential data loss if file reading was not finished when file was removed.
   #close_removed: false
 
+  # Closes the file handler as soon as the harvesters reaches then end of the file.
```

Comment:
@ruflin needs rephrase: Closes the file handler as soon as the harvester reaches <the> end of the file